Extensions/README.md

```diff
@@ -1,15 +1,16 @@
-# Hurl Extension
+# Hurl browser extension
 
 This browser extension uses Native Messaging API to communicate with the Hurl Application.
 
 ## Installation
 
 To use it in Chrome, follow the steps below:
```

Comment:
These steps seem to be seriously outdated?

Reply:
I updated them recently. If anything, I might have missed on adding a step in middle.

Reply:
Maybe it's me, but I can't even find Developer mode in Chrome :(

Reply:
Sorry. I can't even find Dev Mode. I have Dev Tools, but nowhere do can I turn on a _mode_ and the few things I find on the internet aren't helping.

Reply:
I think I know why I can't find Dev Mode - I already have it on 😆